packages/core/src/lib/lit-core.ts

```diff
@@ -944,13 +944,13 @@ export class LitCore {
       throwError({
         ...res.error,
         message:
-          res.error.message ||
+          `Message: ${res.error.message}.  Details: ${res.error.details}` ||
           'You are not authorized to access to this content',
```

Comment:
Now the right-side of this new `||` will never be executed because the left one always resolves to a truthy string. Even when `res.error.message` is not defined

It seems that the not authorized message should be part of the `if` conditions right above, not a fallback here. I would make this change but also remove the right side of this `||`